app/javascript/player/index.jsx

```diff
@@ -67,8 +67,13 @@ class Player extends React.Component<Props, { playing: boolean }> {
   render () {
     const { song } = this.props
     const { playing } = this.state
+    
+    const pStyle = {
```

Comment:
Can you use styled components for this instead of hash styles? https://www.styled-components.com/docs/basics#adapting-based-on-props